SwahiliBox/swahilibox-events-api
# Create database models

#### What does this PR do?
Adds database models for the application

#### Description of Task to be completed?
this Pr implements this issue #14 

#### How should this be manually tested?
I created one endpoint, just a dummy endpoint, `/users` that should return all users in the database.
so if you add users manually in the database, users table, this endpoint should return those users and that means your database is correctly setup.
#### Any background context you want to provide?
The changes so far are for review. When all the feedback is given and all is well then I'll add migration files so the PR can be ready to be merged


### Screenshots?

#### users table structure
<img width="656" alt="Screen Shot 2019-06-05 at 23 31 57" src="https://user-images.githubusercontent.com/17777137/59019428-b65fbe80-8850-11e9-8808-5055961c33bd.png">

#### roles table structure
<img width="731" alt="Screen Shot 2019-06-05 at 23 31 28" src="https://user-images.githubusercontent.com/17777137/59019470-cd061580-8850-11e9-89dd-1aa50ec139e1.png">

#### events table structure
<img width="696" alt="Screen Shot 2019-06-05 at 23 30 10" src="https://user-images.githubusercontent.com/17777137/59019524-e1e2a900-8850-11e9-827e-3d0648d7520d.png">

#### rsvps table structure
<img width="729" alt="Screen Shot 2019-06-05 at 23 29 27" src="https://user-images.githubusercontent.com/17777137/59019551-eeff9800-8850-11e9-9712-c0a5319da4e2.png">
